libs/backend-apisix/e2e/support/utils.ts

```diff
@@ -91,6 +91,16 @@ export const deleteEvent = (
     : undefined,
 });
 
+export const overrideEventResourceId = (
```

Comment:
https://github.com/api7/adc/pull/203/files#diff-af2b7c68fc5660ece2df03f69e1ff4bb37ac1e2b28f5a11549546e1a10e773b0R105

should we only keep one function?

Reply:
They are in different packages and have difficulties in referencing each other, which requires another separate package to place the public functions; and the backends should not depend on each other, so they are kept first.